v-next/hardhat/templates/02-mocha-ethers/contracts/Rocket.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.24;
+
+contract Rocket {
```

Comment:
I added the `Rocket` contract and the Apollo module to the Mocha/Ethers template to enable me to test both deployment and the returned types.

I am happy to remove this, or to alter it to better fit our getting started vision.